Server/mods/deathmatch/logic/luadefs/CLuaAccountDefs.h

```diff
@@ -48,4 +48,5 @@ class CLuaAccountDefs : public CLuaDefs
     LUA_DECLARE(SetAccountPassword);
     LUA_DECLARE(SetAccountData);
     LUA_DECLARE(CopyAccountData);
+    static bool SetAccountSerial(std::string name, std::string serial) noexcept;
```

Comment:
I think it would be better to use `CAccount` since the other functions use it as well
```suggestion
static bool SetAccountSerial(CAccount* account, std::string serial) noexcept;
```

Reply:
adjusted 790ffa2